src/Core/Checkout/Document/DocumentConfiguration.php

```diff
@@ -145,7 +146,7 @@ class DocumentConfiguration extends Struct
     /**
      * @var string|null
      *
-     * @deprecated tag:v6.7.0 - Will be natively typed
+     * @deprecated tag:v6.7.0 - Will be splitted up into single fields. companyStreet, companyZipcode, companyCity, companyCountry
```

Comment:
So everyone understands that this field will be removed:
```suggestion
     * @deprecated tag:v6.7.0 - Will be replaced by single fields: companyStreet, companyZipcode, companyCity, companyCountry
```